src/factories/postburn/PostBurnMCParticles_factory.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2024, Alex Jentsch, Jihee Kim, Brian Page
+// Subject to the terms in the LICENSE file found in the top-level directory.
+//
+
+#include "algorithms/postburn/PostBurn.h"
+#include "algorithms/postburn/PostBurnConfig.h"
+
+// Event Model related classes
+#include <edm4eic/MCRecoParticleAssociationCollection.h>
+#include <edm4eic/ReconstructedParticleCollection.h>
+#include <edm4hep/MCParticleCollection.h>
+
+#include "extensions/jana/JOmniFactory.h"
+
+namespace eicrecon {
+
+class PostBurnMCParticles_factory :
+    public JOmniFactory<PostBurnMCParticles_factory, PostBurnConfig> {
+
+public:
+    using AlgoT = eicrecon::PostBurn;
+private:
+    std::unique_ptr<AlgoT> m_algo;
+
+    PodioInput<edm4hep::MCParticle> m_mcparts_input {this};
+        PodioInput<edm4eic::ReconstructedParticle> m_recoparts_input {this};
+        PodioInput<edm4eic::MCRecoParticleAssociation> m_recoparts_assoc_input {this};
+    PodioOutput<edm4hep::MCParticle> m_postburn_output {this};
```

Comment:
Fix the alignment due to use of tabs.

Reply:
Everywhere else too...

Reply:
I think I got them all.

Reply:
One thing you can do (in general) is run `clang-format -i src/factories/postburn/PostBurnMCParticles_factory.*` etc. It will format everything in line with the rest of the code, or at least new and modified code block (we avoid running this on the entire code base since it causes many changes that don't mean much).